File: Memento/Python/Juego.py
class Juego:
    def __init__(self):
        self._nombre = ''
        self._checkPoint = 0
    
    def getCheckPoint(self):
        return self._checkPoint
    
    def setCheckPoint(self, checkPoint):
        self._checkPoint = checkPoint
    
    def getNombre(self):
        return self._nombre
    
    def setNombre(self, nombre):
        self._nombre = nombre
    
    def toString(self):
        return "Juego [nombre " + self._nombre + " checkpoint = " + str(self._checkPoint) + "]"

File: Memento/Java/Juego.java
public class Juego{
    private String nombre;
    private int checkPoint;

    public int getCheckPoint()
    {
        return this.checkPoint;
    }

    public void setCheckPoint(int checkPoint)
    {
        this.checkPoint = checkPoint;
    }

    public String getNombre()
    {
        return this.nombre; 
    }

    public void setNombre(String nombre)
    {
        this.nombre = nombre;
    }

    @Override
    public String toString()
    {
        return "Juego [nombre " + nombre + " checkpoint= " + checkPoint + "]";
    }
}